cloudslang-entities/src/main/java/io/cloudslang/lang/entities/constants/FileConstants.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * (c) Copyright 2016 Hewlett-Packard Development Company, L.P.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License v2.0 which accompany this distribution.
+ *
+ * The Apache License is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.cloudslang.lang.entities.constants;
+
+/**
+ * @author Bonczidai Levente
+ * @since 9/26/2016
+ */
+public class FileConstants {
```

Comment:
This constant can be replaced by FileNameUtils.EXTENSION_SEPARATOR_STR from commons io, maybe there is no need for this class then